### ISSUE TITLE
ISO-8859-15 character set not supported

### ISSUE THREAD

**Issue by mjk-gh** [jesseduffield/lazygit#1336]
**Describe the bug**
My character set, ISO-8859-15 (Latin9), is not supported.

**To Reproduce**
```
$ git init test
Initialized empty Git repository in /tmp/test/.git/
$ lazygit
2021/05/26 15:22:01 An error occurred! Please create an issue at: https://github.com/jesseduffield/lazygit/issues

*errors.errorString character set not supported
/home/runner/work/lazygit/lazygit/main.go:134 (0xa2f7fb)
/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/proc.go:225 (0x4389d6)
/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/asm_amd64.s:1371 (0x46c621)
$ echo $?
1
$ echo $LC_CTYPE
en_US.ISO-8859-15
$ echo $LANG
en_US.ISO-8859-15
$
```

**Expected behavior**
lazygit should work without problems.

From what I see on the screenshots/casts, all characters could be replaced by even 7-bit ASCII variants:
- up/down arrow in status panel: ^/v or u/d
- "left/right/up/down arrow: navigate" in bottom line: "left/right/up/down: navigate" or "cursor keys: navigate"
- line drawing characters: Either use "+-|" or use ncurses extended charset, which is supposedly compatible to everything:
URL: http://melvilletheatre.com/articles/ncurses-extended-characters/index.html
"Extended characters are designed to be portable between terminals, and should generate usable output from anything that you can run NCURSES on."

**Desktop:**
 - OS: Debian GNU Linux (amd64)
 - Lazygit Version v0.28.1 from https://github.com/jesseduffield/lazygit/releases
 
**References**
Might be related to #24 and #422.

**Comment by jesseduffield** [comment]
@mjk-gh could you see if this PR fixes the issue? https://github.com/jesseduffield/lazygit/pull/1490

**Comment by mjk-gh** [comment]
Seems to work mostly!

Looks like this:

![LC_CTYPE__en_US ISO-8859-15](https://user-images.githubusercontent.com/6675160/134810336-c15fe236-8427-486e-9514-420788f45723.png)

Test with different user:

![openbox-bash-negative-latin9](https://user-images.githubusercontent.com/6675160/134811902-ebabf446-6a54-4f84-9cec-9fa10aa5381c.png)

Test with different user, Latin1 (ISO-8859-1):

![openbox-bash-negative-latin1](https://user-images.githubusercontent.com/6675160/134811905-8b1088b8-a4d7-4b99-9e0d-0bfe6ee0c9bc.png)

Test with different user, UTF8:

![openbox-bash-negative-utf8](https://user-images.githubusercontent.com/6675160/134811961-e0bc212f-6064-46d6-8a15-400df1cdac24.png)

The only two noticeable problems on these screenshots:
- Top left and right corner of collapsed/backgrounded panels have a question mark instead of a horizontal line drawing character (non-UTF8 screenshots only)
- The help text in the bottommost line gets partially overwritten by "Donate  Ask Question" (all screenshots)

The double quote-mark left of "init-this-repo.sh" is probably not a character conversion artifact, but means "untracked", I guess -- have not gotten around to really test lazygit because of the previous unability to start it at all.

**Comment by jesseduffield** [comment]
I reproduced that first issue locally and I've resolved it by just using spaces instead of those edge characters (given that using direct hyphens looks weird and tcell won't let me use the other hyphen-y characters as fallbacks).
The second issue you mention is actually normal (arguably the blue text should get priority though).

I've gone ahead and merged that PR. Lemme know if there are any more issues you come across and I'll re-open this issue :)
